components/LanguageFormField/LanguageFormField.tsx

```diff
@@ -97,13 +99,13 @@ const LanguageFormField: React.FC<Props> = ({ languages, languageRelations, refe
 
       <div className="lang-level-select">
         <Select
-          placeholder={'Select a language...'}
+          placeholder={t('profile.languages.languageSelectPlaceholder')}
           options={langOptions}
           value={selectedLangId}
           onChange={setSelectedLangId}
         />
         <Select
-          placeholder={'Select a level...'}
+          placeholder={t('profile.languages.levelSelectPlaceholder')}
           options={levelOptions}
```

Comment:
The options for languages levels needs to get translated too. It's kind of a pain since they're in a function but you can either pass in the `t` function and let `fullLevelName` call it, or return a translation key from `fullLevelName` and call `t` outside..